contentcuration/contentcuration/models.py

```diff
@@ -695,6 +733,39 @@ class Channel(models.Model):
         "version",
     ])
 
+    @classmethod
+    def filter_edit_queryset(cls, queryset, user):
+        user_id = not user.is_anonymous() and user.id
```

Comment:
This same exact line is repeated below

Reply:
So good I did it twice? Copy paste error - thanks.